clarity-ext-scripts/clarity_ext_scripts/covid/partner_api_client.py

```diff
@@ -210,37 +210,48 @@ class PartnerAPIV7Client(object):
     """
 
     def __init__(self, test_partner_base_url, test_partner_user, test_partner_password,
-                 integration_test_mode=False, integration_test_should_fail=0):
```

Comment:
I'm thinking now that, if we have time, it wouldn't hurt to make a subclass of this client that actually supports integration tests, and have the subclass entirely void of knowledge about testing. But given the tight timeframe, I think this is also fine!

Reply:
This would though have the benefit of simplifying the client.

Reply:
I think we reserve this for later, since right now I've not had time to implement the integration testing mode...

Reply:
I.e. it is not there in the actual methods, but only vestigial from my copy-paste of the v6 client.

Reply:
Right!